docker-compose.yml

```diff
@@ -1,100 +1,69 @@
 version: "2"
 
 services:
-  resolvable:
-    image: mgood/resolvable
-    volumes:
-      - "/var/run/docker.sock:/tmp/docker.sock"
-      - "/etc/resolv.conf:/tmp/resolv.conf"
-
```

Comment:
Sorry if it wasn't very clear, but in this PR docker-compose.yml is the production version, hence why I've removed these bits and they would be re-introduced in docker-compose.dev.yml. Also most of the code changes are just hacky at this point, I should have made that clearer in the PR